src/luarocks/cmd/build.lua

```diff
@@ -30,6 +30,7 @@ function cmd_build.add_to_parser(parser)
       :args("?")
 
    cmd:flag("--only-deps", "Installs only the dependencies of the rock.")
+   cmd:flag("--no-doc", "Installs the rock without its documentation.")
```

Comment:
Should this be part of `make.cmd_options`?

Reply:
`make` didn't previously support `--no-doc` but it probably should. I'll have to move `remove_doc_dir` to util.